_pytest/python.py

```diff
@@ -131,6 +131,7 @@ def pytest_namespace():
     raises.Exception = pytest.fail.Exception
     return {
         'raises': raises,
+        'not_raises': NotRaisesContext,
```

Comment:
I liked `not_raises` due to its similarity to `raises`, but I don't have a strong opinion on it. @The-Compiler do you mind changing to `wont_raise`?